.gitignore

```diff
@@ -145,7 +145,9 @@ php/ext/google/protobuf/Makefile.objects
 php/ext/google/protobuf/acinclude.m4
 php/ext/google/protobuf/build/
 php/ext/google/protobuf/config.h
+php/ext/google/protobuf/config.h.in~
```

Comment:
The trailing `~` looks odd - is that meant to be there?

Reply:
+1. Is this change expected?

Reply:
Yes, I don't know why but compiling php7.3 code always have that.